config.yml.default

```diff
@@ -1,131 +1,131 @@
-#   go_commands:             # Additional options to pass to the UCI go command.
-#     nodes: 1               # Search so many nodes only.
-#     depth: 5               # Search depth ply only.
-#     movetime: 1000         # Integer. Search exactly movetime milliseconds.
-
-# xboard_options:            # Arbitrary XBoard options passed to the engine.
+  uci_options:                     # Arbitrary UCI options passed to the engine.
+    Move Overhead: 500             # Increase if your bot flags games too often.
+    Threads: 4                     # Max CPU threads the engine can use.
+    Hash: 512                      # Max memory (in megabytes) the engine can allocate.
+    SyzygyPath: "./syzygy/"        # Paths to Syzygy endgame tablebases that the engine reads.
+#   go_commands:                   # Additional options to pass to the UCI go command.
+#     nodes: 1                     # Search so many nodes only.
+#     depth: 5                     # Search depth ply only.
+#     movetime: 1000               # Integer. Search exactly movetime milliseconds.
+
+# xboard_options:                  # Arbitrary XBoard options passed to the engine.
 #   cores: "4"
 #   memory: "4096"
-#   egtpath:                 # Directory containing egtb (endgame tablabases), relative to this project. For 'xboard' engines.
+#   egtpath:                       # Directory containing egtb (endgame tablabases), relative to this project. For 'xboard' engines.
 #     gaviota: "Gaviota path"
 #     nalimov: "Nalimov Path"
 #     scorpio: "Scorpio Path"
 #     syzygy: "Syzygy Path"
-#   go_commands:             # Additional options to pass to the XBoard go command.
-#     depth: 5               # Search depth ply only.
+#   go_commands:                   # Additional options to pass to the XBoard go command.
+#     depth: 5                     # Search depth ply only.
 #     Do note that the go commands 'movetime' and 'nodes' are invalid and may cause bad time management for XBoard engines.
 
-  silence_stderr: false      # Some engines (yes you, Leela) are very noisy.
+  silence_stderr: false            # Some engines (yes you, Leela) are very noisy.
 
-abort_time: 20               # Time to abort a game in seconds when there is no activity.
-fake_think_time: false       # Artificially slow down the bot to pretend like it's thinking.
-rate_limiting_delay: 0       # Time (in ms) to delay after sending a move to prevent "Too Many Requests" errors.
-move_overhead: 2000          # Increase if your bot flags games too often.
+abort_time: 30                     # Time to abort a game in seconds when there is no activity.
+fake_think_time: false             # Artificially slow down the bot to pretend like it's thinking.
+rate_limiting_delay: 0             # Time (in ms) to delay after sending a move to prevent "Too Many Requests" errors.
+move_overhead: 1000                # Increase if your bot flags games too often.
```

Comment:
I prefer 2000.

Reply:
In some time controls, 2000 can lead to stupid blunders, that is why I have set it to 1000.

Reply:
I prefer the original configuration of `Move Overhead` 100 and `move_overhead` 2000, than the changes that you have made. `move_overhead` decreases the initial time to avoid timeouts.

Reply:
Changed.